fleet_management_api/script_args/configs.py

```
@@ -1,18 +1,36 @@
 from __future__ import annotations
-from typing import Optional, Any
+from typing import Optional, Any, Literal
```

Ruff

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)